tests/Integration/Core/Test_MySQLDB.py

```diff
@@ -230,8 +230,8 @@ def test_getDistinctAttributeValues(name, fields, requiredFields, values, table,
         ),
         (table, allFields, genVal2(), name, {"older": "UTC_TIMESTAMP()", "timeStamp": "Time"}, 2, True),
         (table, allFields, genVal2(), name, {"newer": "UTC_TIMESTAMP()", "timeStamp": "Time"}, 0, True),
-        (table, allFields, genVal2(), name, {"older": Time.toString, "timeStamp": "Time"}, 2, True),
-        (table, allFields, genVal2(), name, {"newer": Time.toString, "timeStamp": "Time"}, 0, True),
+        (table, allFields, genVal2(), name, {"older": TimeUtilities.toString(), "timeStamp": "Time"}, 2, True),
+        (table, allFields, genVal2(), name, {"newer": TimeUtilities.toString(), "timeStamp": "Time"}, 0, True),
```

Comment:
It is not the same

Reply:
I know, that is the source of error in integration. Have tried with `TimeUtilities.toString` too but it's still failing, even though it should return the same value as the old `Time.toString`